test/local/senders/email.js

```diff
@@ -159,6 +162,8 @@ describe(
               } else {
                 assert.equal(templateName, type)
               }
+
+              assert.equal(templateVersion, TEMPLATE_VERSIONS[templateName], 'template version is correct')
```

Comment:
Wonder if there is an easy way to test that it uses default values when template is not found.